base/linalg/symmetric.jl

```diff
@@ -56,6 +56,12 @@ end
 ctranspose(A::Hermitian) = A
 trace(A::Hermitian) = real(trace(A.data))
 
+#tril/triu
+tril(A::Hermitian,k::Integer=0) = tril(A.data,k)
```

Comment:
I don't think these are right. Aren't half the elements in `A.data` potentially garbage?

```
julia> A = rand(4,4)
4x4 Array{Float64,2}:
 0.992492  0.88037   0.40931   0.311544
 0.623913  0.93559   0.310044  0.598238
 0.393736  0.051534  0.103072  0.74386
 0.747731  0.684462  0.813315  0.0329609

julia> Symmetric(A)
tril(4x4 Base.LinAlg.Symmetric{Float64,Array{Float64,2}}:
 0.992492  0.88037   0.40931   0.311544
 0.88037   0.93559   0.310044  0.598238
 0.40931   0.310044  0.103072  0.74386
 0.311544  0.598238  0.74386   0.0329609

julia> tril(Symmetric(A))
4x4 Array{Float64,2}:
 0.992492  0.0       0.0       0.0
 0.623913  0.93559   0.0       0.0
 0.393736  0.051534  0.103072  0.0
 0.747731  0.684462  0.813315  0.0329609

julia> full(Symmetric(A))
4x4 Array{Float64,2}:
 0.992492  0.88037   0.40931   0.311544
 0.88037   0.93559   0.310044  0.598238
 0.40931   0.310044  0.103072  0.74386
 0.311544  0.598238  0.74386   0.0329609

julia> tril(full(Symmetric(A)))
4x4 Array{Float64,2}:
 0.992492  0.0       0.0       0.0
 0.88037   0.93559   0.0       0.0
 0.40931   0.310044  0.103072  0.0
 0.311544  0.598238  0.74386   0.0329609
```